### PR TITLE
[bitnami/mongodb] Configurable admin/root user name

### PR DESCRIPTION
**Pull request by axdotl**
**Description of the change**

With this change, the name becomes configurable for the admin user.

**Benefits**

Some users prefer to name the user `admin` (or completely different) instead of `root`.

**Possible drawbacks**

None known

**Applicable issues**

  - fixes #

**Additional information**

**Checklist** 

- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. [bitnami/<name_of_the_chart>])

### CODE REVIEW

**Comment by rafariossaa** [comment]
Hi,
  This LGTM but let me bring in other team mate to have a look into it. 
  @juan131 Could you take a look into this ?

File: bitnami/mongodb/values.yaml [hunk 1]
```diff
@@ -133,6 +133,9 @@ auth:
   ## ref: https://docs.mongodb.com/manual/tutorial/enable-authentication/
   ##
   enabled: true
+  ## @param auth.rootPassword MongoDB&reg; root user
+  ##
+  rootUser: "root"
```
Comment (juan131) [inline bitnami/mongodb/values.yaml:138]: Double-quotes aren't necessary.
```suggestion
  rootUser: root
```

**Review by juan131 — CHANGES_REQUESTED** [review]
Hi @axdotl 

Sorry for the late review, I missed this.

---

Does this PR has an associated PR to modify the [Bitnami MongoDB container](https://github.com/bitnami/bitnami-docker-mongodb)? AFAIK, this container doesn't support any `MONGODB_ROOT_USER` env. variable (see [mongodb-env.sh](https://github.com/bitnami/bitnami-docker-mongodb/blob/master/5.0/debian-10/rootfs/opt/bitnami/scripts/mongodb-env.sh)) and assumes the user is "root", see:

- https://github.com/bitnami/bitnami-docker-mongodb/blob/master/5.0/debian-10/rootfs/opt/bitnami/scripts/libmongodb.sh#L506

**Comment by ricoberger** [comment]
Hi @juan131, thanks for your feedback. I created a PR in the [bitnami/bitnami-docker-mongodb](https://github.com/bitnami/bitnami-docker-mongodb), which adds the `MONGODB_ROOT_USER` to the Docker images: https://github.com/bitnami/bitnami-docker-mongodb/pull/291.

**Comment by rafariossaa** [comment]
LGTM.

**Pull request merged**